doc/specification.rst

```diff
@@ -406,11 +408,12 @@ GroundTruth Initialization Parameters
 - ``OSMPGroundTruthInit`` MUST be encoded as ``osi3::GroundTruth`` (see the OSI
   specification documentation for more details).
 
-- ``OSMPGroundTruthInit`` MUST contain all static data (e.g. roads) encountered
-  by the model during a simulation run. Any dynamic data (e.g. MovingObjects)
-  it contains MUST NOT be used and has no specified semantics.
+- ``OSMPGroundTruthInit`` MUST contain all data which is requested by the model 
+  in ``OSMPGroundTruthInitConfiguration``but MUST NOT contain more data. The 
+  model is expected to be able to concatenate data during runtime based on its 
+  request. 
 
-- The Ids of objects in ``OSMPGroundTruthInit`` MUST be identical to the Ids of
+- DEPRECATED: The Ids of objects in ``OSMPGroundTruthInit`` MUST be identical to the Ids of
   the same objects contained in later ``OSMPSensorViewIn`` or other input data.
```

Comment:
Not sure if this can be deprecated. I wrote this originally (but probably not clearly enough) because dynamic objects can reference static objects. So a MovingObject can have assigned lanes, but the SensorView will not contain these lanes, since they are static. Probably should be rewritten to say that, and to mention that SensorViews may be incomplete on their own, if used together with GroundTruthInit (e.g. currently osi-validator complains on our SensorView messages, since it references lanes which are only in the GroundTruthInit), but that SensorView + GroundTruthInit must be consistent.